examples/.env

```diff
@@ -52,6 +52,11 @@ OPENROUTER_KEY=
 ELEVEN_LABS_URL=https://api.elevenlabs.io/v1
 ELEVEN_LABS_API_KEY=
 
+# For using Cartesia
+CARTESIA_URL=https://api.cartesia.ai
+CARTESIA_API_KEY=
+CARTESIA_API_VERSION=2025-04-16
```

Comment:
please keep in mind that we only need env vars for stuff users need to configure - does the URL or API_VERSION change?

Reply:
The `version` is a list on the API so you can use the latest or previous ones, the URL should not change over time (no versioning on it), I'll remove the URL from here.